pkg/config/types/config.go

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
> `pkg/config/types/config.go`

Maybe just `pkg/config/config.go` ?

Reply:
And then `NewConfig` can be renamed to `New`

Reply:
Updated